src/view/com/util/forms/PostDropdownBtn.tsx

```diff
@@ -139,7 +141,7 @@ let PostDropdownBtn = ({
       label: isWeb ? _(msg`Copy link to post`) : _(msg`Share`),
       onPress() {
         const url = toShareUrl(href)
-        shareUrl(url)
+        ShareUrl(url)
```

Comment:
Why was this function capitlalized? Usually that is reserved for React Components.

Reply:
It was reverted, thanks.